packages/rocketchat-api/server/v1/settings.js

```diff
@@ -29,6 +29,30 @@ RocketChat.API.v1.addRoute('settings.public', { authRequired: false }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('settings.oauth', { authRequired: false }, {
+	get() {
+		const mountOAuthServices = () => {
+			const oAuthServicesEnabled = ServiceConfiguration.configurations.find({}).fetch();
+
+			return oAuthServicesEnabled.map((service) => {
+				return {
+					id: service._id,
+					name: service.service,
+					appId: service.appId || service.clientId,
+					buttonLabelText: service.buttonLabelText || '',
+					buttonColor: service.buttonColor || '',
+					buttonLabelColor: service.buttonLabelColor || '',
+					custom: Boolean(service.custom)
```

Comment:
Can you explain why you're using `Boolean(service.custom)` here when we don't use it anywhere else?

Reply:
@graywolf336 to convert to Boolean if it is some value considered false by JS.

Reply:
@MarcosSpessatto Please reconsider another way, because if the string is `"false"` then it will result in a value of `true`: https://stackoverflow.com/a/264037

Reply:
Okay, I'm going to change the validation, but I don't agree that the string  'false'  should be considered Boolean, IMHO  'false' is a string and not a boolean, just as 'true' should be considered a string and not a boolean. So in both cases, casting a Boolean must be true. Why should I consider the string 'false'?

Reply:
You misunderstood what I was saying. If the value of `service.custom` is a string and the value is `'false'` then doing `Boolean(service.custom)` will result in `true` being the value instead of the expected `false`.

Reply:
![image](https://user-images.githubusercontent.com/51996/38209683-c479b440-367a-11e8-893b-9b76e1a09575.png)
a non-empty string in javascript evaluates to true, regardless of content of string

Reply:
The question was, why store in the database a Boolean value as a string and not as Boolean, if you expect it to be Boolean, but @graywolf336  already explained that to me. Thanks.